editor/debugger/debug_adapter/debug_adapter_protocol.cpp

```diff
@@ -960,7 +960,8 @@ void DebugAdapterProtocol::on_debug_stack_frame_var(const Array &p_data) {
 
 void DebugAdapterProtocol::on_debug_data(const String &p_msg, const Array &p_data) {
 	// Ignore data that is already handled by DAP
-	if (p_msg == "debug_enter" || p_msg == "debug_exit" || p_msg == "stack_dump" || p_msg == "stack_frame_vars" || p_msg == "stack_frame_var" || p_msg == "output" || p_msg == "request_quit") {
+	if (p_msg == "debug_enter" || p_msg == "debug_exit" || p_msg == "stack_dump" || p_msg == "stack_frame_vars" ||
+			p_msg == "stack_frame_var" || p_msg == "output" || p_msg == "request_quit" || p_msg == "evaluation_return") {
```

Comment:
The `evaluation_return` message doesn't seem to be handled, so it should not be skipped

```suggestion
	if (p_msg == "debug_enter" || p_msg == "debug_exit" || p_msg == "stack_dump" || p_msg == "stack_frame_vars" || p_msg == "stack_frame_var" || p_msg == "output" || p_msg == "request_quit") {
```